README.md

```diff
@@ -1,18 +1,23 @@
 <!--
-(c) Copyright IBM Corp. 2016, 2017
+Copyright (c) 2016, 2017 IBM Corp. and others
 
-This program and the accompanying materials are made available
-under the terms of the Eclipse Public License v1.0 and
-Apache License v2.0 which accompanies this distribution.
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
 
-    The Eclipse Public License is available at
-    http://www.eclipse.org/legal/epl-v10.html
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath 
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
 
-    The Apache License v2.0 is available at
-    http://www.opensource.org/licenses/apache2.0.php
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
 
-Contributors:
-  Multiple authors (IBM Corp.) - initial implementation and documentation
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
```

Comment:
There is another section lower down that still references the older license (lines 93-95: What's the License?)